### PR TITLE
Remove submodules

### PR DESCRIPTION
**Pull request by CSNWEB**

### CODE REVIEW

File: script/console [hunk 1]
```diff
@@ -46,7 +46,7 @@ if [ -n "$1" ]; then
         elif [ "$1" = "apollo-engine" ]; then
             echo "no shell access"
         elif [ "$1" = "campus-frtonend" ] || [ "$1" = "campus" ]; then
```
Comment (hendrikniemann) [inline script/console]: This is a type, right?

File: script/server [hunk 2]
```diff
@@ -10,7 +10,7 @@ cd "$(dirname "$0")/.."
 echo "If you pulled changes running the update script first is strongly recommended!"
 echo
 echo "==> Starting services"
-services="`cat script/services.lst`"
+services="`cat script/z-services.lst`"
```
Comment (hendrikniemann) [inline script/server]: Hide it with `.services.lst`?

File: script/console (hunk 1 above)
Comment (CSNWEB) [reply script/console]: Seems like it 😄

File: script/server (hunk 2 above)
Comment (CSNWEB) [reply script/server]: Yes much better idea, thanks!

**Pull request merged**